BETA_TESTING.md

```diff
@@ -10,18 +10,15 @@ If you want to make code changes, follow [contributing guidance](https://github.
 
 2. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet.
 
-3. Start DevTools:
+3. Clone the repo, get needed Flutter version to local folder and start DevTools:
 
 ```bash
-# Clone the repo:
-git clone https://github.com/flutter/devtools.git;
+git clone https://github.com/flutter/devtools.git
 
-# Get local Flutter of the correct version:
-./devtools/tool/update_flutter_sdk.sh;
+./devtools/tool/update_flutter_sdk.sh
 
-# Start the application:
-cd devtools/packages/devtools_app;
-../../tool/flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true;
+cd devtools/packages/devtools_app
```

Comment:
nit: leaving the semi colons in allows the reader to copy and paste the whole block if they would like to